fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraResourceImpl.java

```diff
@@ -182,6 +181,8 @@
 
     static final String RDF_TYPE_URI = RDF_NAMESPACE + "type";
 
+    private static final List<String> RESTRICTED_PREDICATES = asList(HAS_MEMBER_RELATION.toString(), RDF_TYPE_URI);
```

Comment:
`RESTRICTED_PREDICATES` is not currently used. I am guessing the intention is to prevent using a (In)DirectContainer to add SMTs, which looks like you already have happening in `validateMemberRelation`. Can this be removed?